template.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+class StringBuffer {
```

Comment:
I think `StringBuffer` class here is superfluous. It represents an array of strings basically so I think it's simpler to just use an array of strings.